### PR TITLE
INTG-2060 delete verb is included when processing repeated query prm

### PR DESCRIPTION
**Pull request by MickStanciu**
include "delete" verb when building requests with arrays in query params

### CODE REVIEW

**Review by HarryTranAU — APPROVED** [review]

File: template/action_execute_code.go [hunk 1]
```diff
@@ -69,7 +70,7 @@ func (t *WorkatoTemplate) getExecuteCode(service *gendoc.Service, method *gendoc
 				}
 			}
 
-			if mthd == httpGet && hasRepeatedType(t.messageMap[method.RequestFullType]) {
+			if (mthd == httpGet || mthd == httpDelete) && hasRepeatedType(t.messageMap[method.RequestFullType]) {
```
Comment (BCook98) [inline template/action_execute_code.go:73]: Is there any reason this needs to be restricted to certain HTTP verbs?

Comment (MickStanciu) [reply template/action_execute_code.go:73]: not 100% reason. Just to contain for what we need (GET and DELETE).
I assume POST/PUT/PATCH they usually take BODY and people don't bother with query params.
I can remove that restriction if you want. Will not make a difference (probably ... need to check)

Comment (BCook98) [reply template/action_execute_code.go:73]: Yeah seems weird to have an arbitrary restriction with no real reason. As we are just going to have to remove it later when we do find a use case.

Comment (MickStanciu) [reply template/action_execute_code.go]: fair enough. I removed them

**Review by BCook98 — APPROVED** [review]

**Pull request merged**